### DISCUSSION TITLE
Unbound added to Pi-hole LXC

### DISCUSSION THREAD

**Discussion by 414YogaDog** [tteck/Proxmox#1087] [Ideas]
All the cool kids seem to be using Unbound with their PiHole.
Is there any chance Unbound could be added to the Pi-hole LXC?

**Comment by Vakrehus** [comment]
you should install unbound directly in your pihole-LXC. Works perfectly for me.

https://docs.pi-hole.net/guides/dns/unbound/

**Reply by 414YogaDog** [reply]
Thank you.
